app/controllers/application_controller.rb

```diff
@@ -169,8 +169,11 @@ def initialize_feature_flags
     # This allows to avoid skipping this filter in many places.
     return unless request.env[:current_marketplace]
 
-    FeatureFlagHelper.init(request, Maybe(@current_user).is_admin?.or_else(false),
-                           Maybe(@current_user).is_marketplace_admin?.or_else(false))
+    FeatureFlagHelper.init(community_id: @current_community&.id,
```

Comment:
We are anyways returning unless there is `request.env[:current_marketplace]`, so we shouldn't need to use safe-navigator here.